e2e/config/application.yml

```diff
@@ -0,0 +1,44 @@
+# This file contains application defaults
```

Comment:
I think this should not be commited as it might reveal sensitive data. If we continue executing automated e2e tests, we can create it right before execution.

Reply:
Removed the committed version